app/views/smart_answers/inputs/_salary_question.html.erb

```diff
@@ -8,7 +8,7 @@
   id: "response_amount",
   heading_size: "l",
   width: 10,
-  hint: "in £",
+  suffix: "in £",
```

Comment:
I think maybe this should be using the prefix attribute instead: https://components.publishing.service.gov.uk/component-guide/input/with_prefix

but that's out of scope for this PR - so maybe just leave out the salary questions

Reply:
Thank you. I'll remove that.